pkgs/development/libraries/qt-5/modules/qtwebengine.nix

```diff
@@ -28,11 +29,29 @@
 , postPatch ? ""
 }:
 
+let
+  # qtwebengine expects to find an executable in $PATH which runs on
+  # the build platform yet knows about the host `.pc` files.  Most
+  # configury allows setting $PKG_CONFIG to point to an
+  # arbitrarily-named script which serves this purpose; however QT
+  # insists that it is named `pkg-config` with no target prefix.  So
+  # we re-wrap the host platform's pkg-config.
+  pkg-config-wrapped-without-prefix = stdenv.mkDerivation {
```

Comment:
If you are only running commands in installPhase you probably want to use runCommand instead